packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2464,6 +2464,51 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     });
   }
 
+  ['@test adds isVisible binding when style binding is missing and other bindings exist'](assert) {
+    let assertStyle = (expected) => {
```

Comment:
Copied from https://github.com/emberjs/ember.js/pull/14104/files#diff-c47395a5aaf92aa0a7edbfa812d4e0afR2400

`assertComponentElement` is problematic for the `style` attribute which is prob. why this function was invented by its author for the other test.